app/modbus.mjs

```diff
@@ -230,6 +252,47 @@ export const getDeviceInformation = async (modbusClient) => {
     return deviceInformation
 }
 
+export const getAlarmStatuses = async (modbusClient, onlyActive = true, distinct = true) => {
+    let alarms = []
+
+    if(distinct === true && onlyActive === false) {
+        alarms = Object.assign([], ALARMS);
+    }
+    
+    const startRegister = 385
+    const endRegister = 518
+    const alarmOffset = 7;
+
+    for (let register = startRegister; register <= endRegister; register += alarmOffset) {
+        let result = await mutex.runExclusive(async () => modbusClient.readHoldingRegisters(register, 7))
+        let code = result.data[0]
+        let state = result.data[1]
+
+        if(ALARMS[code] !== undefined && (onlyActive === false || state > 0)) {
```

Comment:
Can you format the file so that there are spaces before the parentheses e.g. here?

Reply:
sure thing